RASP_support/Support.py

```diff
@@ -176,15 +176,17 @@ def zipmap(n, k_vars, func):
 def verify_default_size(default, num_output_vars):
 	assert num_output_vars > 0
 	if num_output_vars == 1:
-		assert not isinstance(
-			default, tuple), "aggregates on functions with single output \
-			should have scalar default"
+		errnote = "aggregates on functions with single output should have" \
+			+ "scalar default"
```

Comment:
oops, can you add a space here to sit between "have" and "scalar" in the string?

Reply:
No problem :)